grim.go

```diff
@@ -117,10 +117,21 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 		if err != nil {
 			return grimErrorf("error extracting hook from message: %v", err)
 		}
+		logger.Print(hook.Describe() + "\n")
```

Comment:
You can just return a string if you want.  The problem with that is that you can't be sure if "" means dont skip or skip with no reason.

So a pointer to a string would work as well.